app.json

```diff
@@ -68,16 +68,16 @@
     "NORTHSTAR_URL": {
       "required": true
     },
-    "PHOENIX_API_VERSION": {
+    "ASHES_API_VERSION": {
       "required": true
     },
-    "PHOENIX_PASSWORD": {
+    "ASHES_PASSWORD": {
       "required": true
     },
-    "PHOENIX_URI": {
+    "ASHES_URI": {
       "required": true
     },
-    "PHOENIX_USERNAME": {
+    "ASHES_USERNAME": {
       "required": true
```

Comment:
@sbsmith86 should I add this now or later when we actually start using Phoenix endpoints (and when we add the ENV variables to Heroku)?

Reply:
@chloealee ahh i think it is fine to leave it as it is! That is a good point.